client/state/themes/actions.js

```diff
@@ -404,6 +405,44 @@ export function clearActivated( siteId ) {
 }
 
 /**
+ * Triggers a network request to install WordPress.com theme on Jetpack site.
+ * After installataion it switches page to the customizer
+ * Requires Jetpack 4.4
+ *
+ * @param  {String}   themeId      WP.com Theme ID
+ * @param  {String}   siteId       Jetpack Site ID
+ * @return {Function}              Action thunk
+ */
+export function installAndTryAndCustomize( themeId, siteId ) {
+	return ( dispatch, getState ) => {
+		return dispatch( installTheme( themeId, siteId ) )
+			.then( () => {
+				const theme = getTheme( getState(), siteId, themeId );
+				if ( ! theme ) {
+					return;
+				}
+				const url = getThemeCustomizeUrl( getState(), theme, siteId );
+				page( url );
+			} );
+	};
+}
+
+/**
+ * thin wrapper arround installAndTryAndCustomize so it can be used int theme action
+ *
+ * @param  {String}   themeId      WP.com Theme ID
+ * @param  {String}   siteId       Jetpack Site ID
+ * @return {Function}              Action thunk
+ */
+export function installWpcomThemeAndTryAndCustomize( themeId, siteId ) {
```

Comment:
For consistency's sake, let's nix the wrapper and add the suffix [in `theme-options`](https://github.com/Automattic/wp-calypso/pull/10126/files#diff-b4173dc1d07244fce42a671d04c5c1caR101) (like [so](https://github.com/Automattic/wp-calypso/blob/a614ba01940357b5e423d1b72cf353946ca7a563/client/my-sites/themes/theme-options.js#L63))